backend/mora/service/detail_writing.py

```diff
@@ -357,10 +379,14 @@ def create():
 
     """
 
+    reqs = flask.request.get_json()
+
+    validated = process_requests(CREATE_VALIDATION_HANDLERS, reqs)
+
     return (
         flask.jsonify(
             process_requests(CREATION_HANDLERS,
-                             flask.request.get_json()),
+                             validated),
```

Comment:
This feels somewhat weird — `validated` reads like a `bool`, but probably represents a preprocessed state or something similar?

Reply:
Yeah, it represents a set of payloads that have been through validation etc. and are ready to be sent to LoRa. I'll think of a better name.